batchlib/analysis/cell_level_analysis.py

```diff
@@ -316,15 +368,7 @@ def save_all_stats(self, in_file, out_file, device):
         with open(out_file[:-6] + 'json', 'w') as fp:
```

Comment:
Again, would be good if we can use `self.write_table` instead. (In case `label_id` doesn't make sense, just set it to `None`).